### PR TITLE
Move t-z converters to utils, and adopt style

### PR DESCRIPTION
**Pull request by MaxThevenet**
This PR proposes to move the `import_from_z` `export_to_z` functions from the `Laser` class to the utils. The arguments of the function complies with others in the utils (namely that it does not depend on advanced types defined in lasy, only on the `Grid` class).

### CODE REVIEW

File: lasy/utils/laser_utils.py [hunk 1]
```diff
@@ -459,3 +467,183 @@ def create_grid(array, axes, dim):
...
+    ----------
+    dim : string
+        Dimensionality of the array. Options are:
+        - 'xyt': The laser pulse is represented on a 3D grid:
+                 Cartesian (x,y) transversely, and temporal (t) longitudinally.
+        - 'rt' : The laser pulse is represented on a 2D grid:
+                 Cylindrical (r) transversely, and temporal (t) longitudinally.
+
+    grid : a Grid object.
+        It contains a ndarrays (V/m) with
```
Comment (AngelFP) [inline lasy/utils/laser_utils.py:480]: `an ndarray`

File: lasy/utils/laser_utils.py [hunk 2]
```diff
@@ -459,3 +467,183 @@ def create_grid(array, axes, dim):
...
+        Spatial `z` axis along which the field should be reconstructed.
+        If not provided, `z_axis = c * t_axis` is considered.
+
+    z0 : scalar (optional)
+        Position from which the field is produced (emitted).
+
+    t0 : scalar (optional)
+        Moment of time at which the field is produced.
+
+    backend : string (optional)
```
Comment (AngelFP) [inline lasy/utils/laser_utils.py:497]: could we mention here a list of the possible values?

File: lasy/utils/laser_utils.py [hunk 3]
```diff
@@ -4,6 +4,14 @@
...
 from skimage.restoration import unwrap_phase
 
+try:
+    from axiprop.lib import PropagatorFFT2, PropagatorResampling
+    from axiprop.containers import ScalarFieldEnvelope
+
+    axiprop_installed = True
+except ImportError:
+    axiprop_installed = False
+
```
Comment (AngelFP) [inline lasy/utils/laser_utils.py]: Should we wait until a decision about #164 is made to add these checks?

File: lasy/utils/laser_utils.py [hunk 4]
```diff
@@ -459,3 +467,183 @@ def create_grid(array, axes, dim):
...
+    ----------
+    dim : string
+        Dimensionality of the array. Options are:
+        - 'xyt': The laser pulse is represented on a 3D grid:
+                 Cartesian (x,y) transversely, and temporal (t) longitudinally.
+        - 'rt' : The laser pulse is represented on a 2D grid:
+                 Cylindrical (r) transversely, and temporal (t) longitudinally.
+
+    grid : a Grid object.
+        It contains a ndarrays (V/m) with
```
Comment (AngelFP) [inline lasy/utils/laser_utils.py]: Same typo as in the other function: `an ndarray`

File: lasy/utils/laser_utils.py (hunk 1 above)
Comment (MaxThevenet) [reply lasy/utils/laser_utils.py:480]: thanks, fixed

File: lasy/utils/laser_utils.py (hunk 2 above)
Comment (MaxThevenet) [reply lasy/utils/laser_utils.py:497]: I am not sure we want to support this at the moment, so I just redirected to Axiprop in a slightly more specific way than was before.

File: lasy/utils/laser_utils.py (hunk 3 above)
Comment (MaxThevenet) [reply lasy/utils/laser_utils.py]: True, I just removed the checks. We'll make it consistent with #164 when a decision is made.

**Comment by MaxThevenet** [comment]
Thanks for your review! I just incorporated your comments

**Review by AngelFP — APPROVED** [review]

**Pull request merged**